deploy/operator.yaml

```diff
@@ -14,7 +14,7 @@ spec:
     spec:
       containers:
         - name: keycloak-operator
-          image: quay.io/integreatly/keycloak-operator:master
```

Comment:
@philbrookes should I change this value on master?

Reply:
@maleck13 WDYT? I think master should point at master these days, but it's been changing a lot recently, so I'm not sure what the current attitude is